eu.octanne.McBoyard.yml

```diff
@@ -0,0 +1,64 @@
+app-id: eu.octanne.McBoyard
+separate-locales: false
+
+base: io.atom.electron.BaseApp
```

Comment:
Why do you need electron here?

Reply:
oh yes it's a mistake I don't need electron